avm/res/web/static-site/main.bicep

```diff
@@ -301,7 +319,23 @@ module staticSite_privateEndpoints 'br/public:avm/res/network/private-endpoint:0
         'Full'
       ).location
       lock: privateEndpoint.?lock ?? lock
-      privateDnsZoneGroup: privateEndpoint.?privateDnsZoneGroup
+      // privateDnsZoneGroup: privateEndpoint.?privateDnsZoneGroup
```

Comment:
This looks like the change is still tested. Should the PR be in draft?

Reply:
No, I just hadn't removed this, my bad

Reply:
udpated